packages/nx-prisma/test-suite.ts

```diff
@@ -0,0 +1,11 @@
+import { generateSuite } from './src/executors/generate/executor.spec';
+import { migrationsSuite } from './src/executors/migrations/executor.spec';
+import { resetSuite } from './src/executors/reset/executor.spec';
+import { seedSuite } from './src/executors/seed/executor.spec';
+
+describe('Run Test', () => {
+  migrationsSuite();
+  generateSuite();
+  seedSuite();
+  resetSuite();
+});
```

Comment:
The order which the test runs is important as you cannot `seed` or `reset` until you have `generate` so this pattern was used to assure the test order.